packages/quantic/force-app/examples/main/lwc/exampleQuanticStandaloneSearchBox/exampleQuanticStandaloneSearchBox.js

```diff
@@ -0,0 +1,45 @@
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticStandaloneSearchBox extends LightningElement {
+  @api engineId = 'quantic-standalone-search-box-engine';
+  @track config = {};
+  isConfigured = false;
+
+  pageTitle = 'Quantic Standalone Search Box';
+  pageDescription = 'The QuanticStandaloneSearchBox component creates a search box with built-in support for query suggestions.';
+  options = [
+    {
+        attribute: 'placeholder',
+        label: 'Placeholder',
+        description: 'The placeholder text to display in the search box input area.',
+        defaultValue: 'Search',
+    },
+    {
+        attribute: 'withoutSubmitButton',
+        label: 'Without submit button',
+        description: 'Whether not to render a submit button.',
+        defaultValue: false,
+    },
+    {
+        attribute: 'numberOfSuggestions',
+        label: 'Number of suggestions',
+        description: 'The maximum number of suggestions to display.',
+        defaultValue: 5,
+    },
+    {
+        attribute: 'redirectUrl',
+        label: 'Redirect url',
+        description: 'The url of the search page to redirect to when a query is made.',
+        defaultValue: '/global-search/%40uri',
```

Comment:
I'm a bit torn between having the actual default value like we have for every other components or use `'/full-search-example'` so that it redirects to an existing page rather than be broken by default. What do you think?

Reply:
it changes nothing in my test, I use `/full-search-example` as custom for redirectUrl and it works 😄

Reply:
that's great! My thinking is that if a human visits the example community then this default value won't be very helpful. But  it's not a critical question to answer at the moment.